### ISSUE TITLE
ctx.meta

### ISSUE THREAD

**Issue by flaviocordova** [berkesa/moleculer-java#16]
In nodejs app-gateway can authenticate user and add current user in ctx.meta (which is added by the authorize method in app-gateway).

Since I can invoke a java action through a nodejs gateway, is there a way to access meta information (specially meta.user) ?

**Comment by berkesa** [comment]
Java and NodeJS implementations transfer the "meta" structure to each other. The main difference is in accessing "meta"; In NodeJS, meta is a parameter of "ctx" (**ctx.meta**), in Java version meta is a parameter of the "params" (**ctx.params.getMeta()**).

```
public Action action = ctx -> {
  Tree meta = ctx.params.getMeta();

  int num = meta.get("key1", 123); // 123 is the default value
  String txt = meta.get("key2.key3", "defaultValue");
  String user = meta.get("user", "");

  Tree roles = meta.get("roles"); // "roles" is a JSON array in this case
  for (Tree role: roles) {
     String roleName = role.get("name", "");
  }
  . . .
};
```

Sending "meta" in response:
```
public Action action = ctx -> {
  Tree rsp = new Tree();
  rsp.put("a", 1);
  rsp.put("b", true);
  rsp.put("c", "xyz");
  rsp.getMeta().put("key", "value");
  return rsp;
};
```

**Comment by berkesa** [comment]
There was an error in the meta-transfer. This bug is fixed in Moleculer-Java version **1.1.2**.
